src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java

```diff
@@ -733,6 +733,47 @@ public void onPlayerCommandPreprocess(PlayerCommandPreprocessEvent event)
                 }
             }
         }
+            if (!TFM_AdminList.isTelnetAdmin(player, true))
```

Comment:
should be if (!TFM_AdminList.isTelnetAdmin(player, true) || !TFM_AdminList.isSeniorAdmin(player, true) || !TFM_AdminList.SERVER_OWNERS.getList().contains(player))